frontend/public/components/monitoring/_monitoring.scss

```diff
@@ -232,18 +232,17 @@ $tooltip-background-color: #151515;
   z-index: 1;
 }
 
-.co-alert-manager-yaml__explanation {
-  margin-bottom: 0;
-}
-
-.co-alert-manager-yaml__form-entry-wrapper {
-  position: relative;
-}
-
-.co-alert-manager-yaml__form co-file-dropzone__textarea {
-  min-height: 400px;
+.btn-edit-alert-routing {
+  margin-bottom: 10px;
 }
 
 .co-alert-manager-config__edit-alert-routing-btn {
   margin-bottom: 10px;
 }
+
+.co-alertmanager-yaml-alert {
+  margin-bottom: 4px;
+  @media (min-width: $screen-sm-min) {
+    margin-bottom: -26px;
```

Comment:
Can you explain why the negative margin is needed?

Reply:
Previously, the Alerts for the YAML Editor appeared between the textarea and buttons:
![image](https://user-images.githubusercontent.com/12733153/70169942-cc342c00-1699-11ea-8f98-eced2306e2e6.png)

The alert states (error & success) were being handled within the YAML Editor and based on saving a k8s resources.  I'm hijacking the YAML Editor and providing my own `save()` function and handing error & success in a higher component, and thus putting the alerts above the textarea.   If I don't adjust the bottom there is a very large gap, which may be minimized if we remove the `view shortcuts` link:
![image](https://user-images.githubusercontent.com/12733153/70170293-6c8a5080-169a-11ea-96d8-9d8dec697df1.png)

I'll hide the `view shortcuts` link and thus probably won't have to use a negative margin.
I could also try to have my custom `save()` throw exceptions to be caught by the YAML editor and thus displayed below the textarea.

Reply:
I wonder if we shouldn't pass this as `children` to the edit yaml component so we can show it in the same spot.

Reply:
Passing the Alerts as children worded great.  -thanks